docs/minecraft/paper.md

```diff
@@ -132,3 +137,13 @@ if (commandManager.hasCapability(CloudBukkitCapabilities.NATIVE_BRIGADIER)) {
 ## Parsers
 
 `cloud-paper` has access to all the parsers from [cloud-bukkit](bukkit.md#parsers).
+
+## Provided Sender Mapper
```

Comment:
Something people are constantly asking for is how to use CSS to restrict commands to players. It might be worth showing how you'd do that using the `PlayerSource` even though it should be obvious if you've read the rest of the docs (but we can assume that they haven't...)

Reply:
changed example to compiled code